application/views/invoice_templates/pdf/InvoicePlane - overdue.php

```diff
@@ -35,23 +35,25 @@
             if ($invoice->client_city) {
                 echo htmlsc($invoice->client_city) . ' ';
             }
-            if ($invoice->client_state) {
-                echo htmlsc($invoice->client_state) . ' ';
-            }
             if ($invoice->client_zip) {
                 echo htmlsc($invoice->client_zip);
             }
             echo '</div>';
         }
+        if ($invoice->client_state) {
+            echo '<div>' . htmlsc($invoice->client_state) . ' ';
+        }
         if ($invoice->client_country) {
-            echo '<div>' . get_country_name(trans('cldr'), $invoice->client_country) . '</div>';
+            echo get_country_name(trans('cldr'), $invoice->client_country) . '</div>';
         }
 
-        echo '<br/>';
-
         if ($invoice->client_phone) {
             echo '<div>' . trans('phone_abbr') . ': ' . htmlsc($invoice->client_phone) . '</div>';
-        } ?>
+        } 
+        if ($invoice->client_mobile) {
+            echo '<div>M: ' . htmlsc($invoice->client_mobile) . '</div>';
+        }
+        ?>
 
```

Comment:
I wouldn't hardcode the M like this

Look at the example on top of your if-statement. Use a translation mobile_abbr